compiler/fm/src/lib.rs

```diff
@@ -94,7 +90,10 @@ impl FileManager {
         self.id_to_path.get(&file_id).unwrap().as_path()
     }
 
-    pub fn find_module(&mut self, anchor: FileId, mod_name: &str) -> Result<FileId, String> {
+    // TODO: This should also ideally not be here, so that the file manager
+    // TODO: does not know about rust modules.
+    // TODO: Ideally this is moved to def_collector_mod and we make this method accept a FileManager
```

Comment:
I can create an issue for this -- its not related to this PR, so I left it as is

Reply:
https://github.com/noir-lang/noir/issues/3799